python_scripts/ensemble.py

```diff
@@ -829,13 +840,13 @@ def plot_decision_function(X, y, clf, sample_weight=None, fit=True, ax=None):
 print(f"Error of the tree: {y_true - y_pred_first_and_second_tree:.3f}")
 
 # %% [markdown]
-# We choose a sample for which only two trees were enough to make the perfect
+# We chose a sample for which only two trees were enough to make the perfect
 # prediction. However, we saw in the previous plot that two trees were not
-# enough to correct the residuals for all samples. In this regard, one need to
-# add several trees in the ensemble to succeed to correct the error.
+# enough to correct the residuals of all samples. Therefore, one needs to
+# add several trees to the ensemble to successfully correct the error.
```

Comment:
Maybe clarify that we are successively fitting the residuals of the previous estimator, i.e., clf1 fits the residuals of clf0, clf2 fits the residuals of clf1 ... Maybe this is obvious from the text, I'm unsure about this nitpick.

Reply:
see https://github.com/lucyleeow/scikit-learn-mooc/pull/1